server/main.py

```diff
@@ -75,6 +75,12 @@ def dev():
         flask.abort(404)
     return flask.render_template('dev.html')
 
+@app.route('/choropleth')
+def choropleth():
+    if os.environ.get('FLASK_ENV') == 'production':
```

Comment:
```suggestion
    # TODO(iancostello): Permit production use after development finishes.
    if os.environ.get('FLASK_ENV') == 'production':
```

The idea is to have todos wherever one has to revisit the code, easing finding places to change among thousands of lines.

Reply:
Added.